app/Http/Controllers/CheckoutController.php

```diff
@@ -0,0 +1,50 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use App\Artwork;
+use Illuminate\Http\Request;
+use Illuminate\Support\Facades\Http;
+
+class CheckoutController extends Controller
+{
+    public function index()
+    {
+        $randomItemsIds = Artwork::all()->random(rand(1, 5))->pluck('id');
+        session(['wishlist.items' => $randomItemsIds]);
+        $wishlistItems = Artwork::whereIn(
+            'id',
+            session('wishlist.items')
+        )->get();
+
+        return view('billing.checkout', compact('wishlistItems'));
+    }
+
+    public function store(Request $request)
+    {
+        $validatedData = $request->validate([
+            'culqi_email' => 'required',
+            'culqi_token' => 'required',
+        ]);
+        $artworks = Artwork::whereIn(
+            'id',
+            session()->pull('wishlist.items')
+        )->get();
```

Comment:
here we should only allow the system to begin the checkout with the items that are available and probably it would be best to let the customer know that the checkout only involves the currently available artworks

Reply:
and of course, change the status of the artworks purchased once the checkout finishes successfully.